source/developers-guide/plugin-quick-start/index.md

```diff
@@ -1043,6 +1043,8 @@ Example:
 The given `label` is the display name of the button.
 You can define an option `handler` as callback for click events.
 
+Visit [Plugin Configuration](https://developers.shopware.com/developers-guide/plugin-configuration/) for further information.
```

Comment:
please write the link without the complete URL like /developers-guide/plugin-configuration/